client/src/locales/en.json

```diff
@@ -30,7 +30,8 @@
     "save": "Save",
     "addSignupQuestion": "Add sign-up question",
     "removeSignupQuestion": "Delete sign-up question",
-    "close": "Close"
+    "close": "Close",
+    "saveInformation": "Save information"
```

Comment:
Varmaan riittäis käyttää "save"-avainta tosta muutama rivi ylempää, sitä on muuallakin käytetty.

Reply:
Muutettu `save`